packages/eas-cli/src/commands/project/init.ts

```diff
@@ -19,14 +19,14 @@ export default class ProjectInit extends EasCommand {
 
     if (exp.extra?.eas?.projectId) {
       Log.error(
-        `app.json is already linked to project with ID: ${chalk.bold(exp.extra?.eas?.projectId)}`
+        `app.json is already linked to app with ID: ${chalk.bold(exp.extra?.eas?.projectId)}`
```

Comment:
renamed `project` to `app`. `app` is the more modern, preferred term, right?

Reply:
We use them somewhat interchangeably all over the place. I think project is still preferred.